config/HELICSConfig.cmake.in

```diff
@@ -124,7 +124,7 @@ find_program(HELICS_APP helics_app
 	)
 	
 find_program(HELICS_BROKER helics_broker 
-	PATHS ${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_BINDIR@
+	PATHS c
```

Comment:
Was this change intentional?

Reply:
no,  I was tinkering with that file before I saw your latest PR and I like yours better so that file should not have changed.  I probably screwed it up,  If you like the changes in the other file just add them to yours and I can delete the branch.